CONTRIBUTING.md

```diff
@@ -244,8 +244,10 @@ If you have an affiliation to the brand you contributing that allows you to spea
 ## Building Website Locally
 
 * Make sure you have [Ruby](https://www.ruby-lang.org/en/downloads/) installed.
-* Make sure you have [Jekyll](https://jekyllrb.com/) installed (using `$ gem install jekyll bundler`). At least version 3.5 is required.
-* Build and run the website locally using `$ jekyll serve`.
+* Make sure you have [Bundler](https://bundler.io/) installed (using `$ gem install bundler`).
+* Set locally the gems installation location for simple-icons website using `$ bundle config set path 'vendor/bundle'`.
```

Comment:
Currently the sentence is a bit awkward. Not sure if _"locally"_ is needed here, so I'd suggest removing it.

```suggestion
* Set the gems installation location for the simple-icons website using `$ bundle config set path 'vendor/bundle'`.
```

If you do want it I suggest

```suggestion
* Set your local gems installation location for the simple-icons website using `$ bundle config set path 'vendor/bundle'`.
```

Reply:
Sure!